Src/stm32wlxx_hal_rcc_ex.c

```diff
@@ -193,6 +193,9 @@ HAL_StatusTypeDef HAL_RCCEx_PeriphCLKConfig(RCC_PeriphCLKInitTypeDef  *PeriphClk
       status = ret;
     }
 
+    /* Disable write access to Backup domain */
+    HAL_PWR_DisableBkUpAccess();
```

Comment:
Such call to a an API from the `PWR` HAL driver shall be done at user application level, if needed, not at `RCC` HAL driver's.